README.md

```diff
@@ -113,13 +113,14 @@ Currently the most reliable way of installing Gatekeeper is to build and install
 
 #### Deploying via Helm ####
 
-A basic Helm v2 template exists in `charts/gatekeeper`. If you have Helm installed and Tiller initialized on your cluster you can deploy via
+A basic Helm chart exists in `charts/gatekeeper`. If you have Helm installed, you can deploy via the following instructions for Helm v3:
+
 ```sh
 helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
-helm install gatekeeper/gatekeeper --devel
+helm install gatekeeper/gatekeeper --generate-name
 ```
 
-Please note that this chart is not compatible with Helm 3 at this time.
+Please note that this chart is compatible with Helm v3 starting with Gatekeeper v3.1.1.
```

Comment:
```suggestion
Please note that this chart is compatible with Helm v3 starting with Gatekeeper v3.1.1.

To check if you have v3.1.1, run the following commands:

```sh
# you may need to update this repo url if you had a reference to the old chart repo
helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts

# pull the latest chart releases
helm repo update

# get all releases
helm search repo gatekeeper --versions --devel 
```
```

Reply:
Reported by #846

Reply:
why is update or search needed here?

Reply:
Added `--force-update` in case someone is using old repo url.